ui/public/locales/en.json

```diff
@@ -1655,7 +1666,7 @@
 "label.rules.file.import.description": "Click or drag rule definitions CSV file to import.",
 "label.rules.file.to.import": "Rule definitions CSV file to import",
 "label.run.proxy.locally": "Run proxy locally",
-"label.running": "Running VMs",
+"label.running": "Running",
```

Comment:
Does this change require adding "VMs" where this label was used?

Reply:
Nice catch. I could only find this label in use at usage dashboard. Let me use another label for that dashboard.
![image](https://user-images.githubusercontent.com/8760112/229107741-139e82a4-9974-41e7-a4ed-9667b62a60b5.png)